File: Python/Examples/playlist.py
import requests

url = "https://api.spotify.com/v1/users/{{user_id}}/playlists"

payload = "{\n  \"name\": \"{{artist_name}} Mix\",\n  \"public\": false\n}"
headers = {
    'accept': "application/json",
    'authorization': "Bearer {{authorization}}",
    'content-type': "application/json"
    }

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)

File: Node/Examples/playlist.js
var http = require("https");

var options = {
  "method": "POST",
  "hostname": [
    "api",
    "spotify",
    "com"
  ],
  "path": [
    "v1",
    "users",
    "{{user_id}}",
    "playlists"
  ],
  "headers": {
    "accept": "application/json",
    "authorization": "Bearer {{authorization}}",
    "content-type": "application/json"
  }
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write(JSON.stringify({ name: '{{artist_name}} Mix', public: false }));
req.end();